modules/ROOT/pages/index.adoc

```diff
@@ -81,8 +83,8 @@ The Object Store Connector operations that extend TTL are:
 |===
 
 
-* The minimum rolling TTL is 1 second.
-* The maximum rolling TTL is unlimited as long as the data continues to be accessed.
+* The minimum rolling TTL is 30 days (2592000 seconds).
+* The maximum rolling TTL is unlimited if you continue to access the data in the last seven days of the 30-day window.
```

Comment:
The original statement is still technically right.. As long as data continues to be accessed, user shouldn't have to worry about TTL.

Reply:
reverted

Reply:
@hannanelson everything else looks good. Just for this statement regarding maximum rolling TTL, I think we can keep the original statement so that it's simple for users to understand. If they want to understand nuances of exactly how the ttl gets extended, they can refer to other sections in the doc where you have this info. What are your thoughts on this?